src/fw-logs/fw-string-formatter.cpp

```diff
@@ -56,7 +56,7 @@ namespace librealsense
 
                 st_regular_exp[2] << "\\{\\b(" << i << "):f\\}";
                 regular_exp[2] = st_regular_exp[2].str();
-                st_replacement[2] << params[i];
+                st_replacement[2] << *reinterpret_cast< const float * >( &params[i] ); // Parse as 4 raw bytes of float
```

Comment:
Can we add some std check of `is_floating_point`?
https://en.cppreference.com/w/cpp/types/is_floating_point

Reply:
I don't think `is_floating_point` will help in this case. `std::isfinite` is more appropriate here (https://en.cppreference.com/w/cpp/numeric/math/isfinite).
If you want I will add a check. If it is not finite, should I print as decimal int, hexadecimal int or throw?

Reply:
Not sure what a thrown will cause here, but maybe we can LOG_ERROR and print as HEX?

Reply:
Done